docs/Presentaciones/02-04-2024.md

```diff
@@ -0,0 +1,150 @@
+el retorno de la inversión en situaciones optimistas, pesimistas y más probables.
+
+Como veis en la gráfica, los costes son un poco mas grandes al principio, en el periodo de desarrollo, pero conforme entramos en producción bajan, a la vez que aumentan los ingresos. Segun hemos calculado
+nuestro servicio seria generaría mas de lo que cuesta a partir de los 30 meses aproximadamente tanto en el escenario optimista como en el más probable, y en el pesimista, que asume un crecimiento
+practicamente nulo de clientes, no parece que vayamos a recuperar la inversión.
+
+## Seguimiento de costes + burndown
+
+Y ya que vemos aqui en los primeros cuatro meses el costo del desarrollo, vamos a hacer un poquito de zoom ahi para ver como vamos:
+
+_Diapositiva con gráfica de costes y burndown_
+
+La linea azul son los costes de los que estamo hablando, aqui representados de forma acumulativa, como vemos en un principio nos encontramabos por encima de los esperado, pero en esta última semana
+se ve un descenso ya que estamos empezando a compensar este exceso de costes y así volver a la linea esperada.
+
+Pero bueno, tampoco es una preocupación porque como vemos con la gráfica rosa, hemos completado mas puntos de historia de lo que esperabamos, lo que significa que estamos avanzo bien y tenemos margen de maniobra
+para corregir estas desviaciones.
+
+_Burndown Sprint 2_
+
+Mirando un poquito más de cerca el avance de este Sprint vemos en azul el burndown de puntos de historia ideal frente al real. Como vemos ibamos un poco atrasados por la mitad del Sprint pero finalmente nos
+recureramos y conseguimos completar casi todas las historias que nos habiamos propuesto. Las que no se ha debido a algun bloqueo que comentaremos mas adelante.
+Cabe destacar que este burndown optimo ha sido actualizado a lo largo del Sprint si se han producido cambios en la planificación o priorización de las actividades.
+
+## Equipo y estado del CA
+
+Continuando con la temática del seguimiento veamos como va el equipo y el estado del CA.
+
+**OPCIONAL, SOLO SI HAY TIEMPO**
+
+Como recordatorio, nosotros medimos en este apartado 5 puntos principales: El cumplimiento minimo de una dedicacion de 10 horas semanales, compensables a trabes de un sistema de compensación de tiempo
+si una semana se echa más y otra menos tiempo, el respeto al horario de disponibilidad establecido, la realizacion de las tareas asignadas dentro de plazos o justificacion de su retraso, y por ultimo
+la correcta aplicación del sistema de avisos.
+
+**FINALIZA OPCIONAL**
+
+A lo largo del proyecto este cumpolimiento ha sido excelente tanto a nivel general como indivual, quitando alguna semana puntual en la que algún miembro hubiera recibido algun aviso por no cumplir con
+la clausula relaciuonada con el cumplimiento de tareas o justificación de retrasos.
+
+Especialmente este Sprint ha sido excepcional sin ninguna incidencia.
+
+_Destacar a los mejores miembros del equipo y a los que más han trabajado_
```

Comment:
The narration of the prototype is missing, although I know that you are in the process of editing the video and that is why this section is not yet here.